netdata-installer.sh

```diff
@@ -157,6 +163,7 @@ USAGE: ${PROGRAM} [options]
   --nightly-channel          Use most recent nightly udpates instead of GitHub releases.
                              This results in more frequent updates.
   --disable-go               Disable installation of go.d.plugin.
+  --disable-mosquitto        Don't build with libmosquitto (this limits Netdata Cloud functionality).
```

Comment:
Doesn't `limit` It `disables it completely` currently and it will not change that soon (in distant future HTTPS fallback might be used).

Reply:
I think the switch should be `--disable-cloud` and it should be used to turn off this functionality now, and any other functionality associated with the cloud in the future. This is what I understood from the discussion on Slack, please correct me if I've summarized the discussion incorrectly. (@cosmix please review)

Reply:
Ah, OK, I had not realized this.

Reply:
For the moment I've just pushed a fix to the option description. Depending on what @cosmix says, I'll update the naming as needed.

Reply:
Quick update - we didn't have any decision at all. It's not clear if we should have either flag or no flag at all. Somebody from product needs to make a decision as it is an important user-facing feature. @cakrit will chase it up and let us know what is decided.

In the mean-time stick to the flag that you had and we will see what the decision is.